SalesforceSDK/Core/Rest/ApiVersionStrings.cs

```diff
@@ -29,7 +29,7 @@ namespace Salesforce.SDK.Rest
 {
     public class ApiVersionStrings
     {
-        public const string VersionNumber = "v31.0";
+        public const string VersionNumber = "v35.0";
```

Comment:
Has ```v35.0``` been rolled out to all prod instances yet? If not, I'd hold off on this, until that happens.